omnisafe/algorithms/on_policy/base/policy_gradient.py

```diff
@@ -81,18 +81,26 @@ def __init__(self, env_id: str, cfgs: Config) -> None:
             f'or increase batch size {self.cfgs.steps_per_epoch}.'
         )
 
-        # set up logger and save configuration to disk
-        self.logger = Logger(exp_name=cfgs.exp_name, data_dir=cfgs.data_dir, seed=cfgs.seed)
-        self.logger.save_config(cfgs.todict())
         # setup actor-critic module
         self.actor_critic = ConstraintActorCritic(
             observation_space=self.env.observation_space,
             action_space=self.env.action_space,
             model_cfgs=cfgs.model_cfgs,
         ).to(self.device)
         self.set_mpi()
-        # set up experience buffer
 
+        # set up logger and save configuration to disk
+        self.logger = Logger(
+            output_dir=cfgs.data_dir,
+            exp_name=cfgs.exp_name,
+            seed=cfgs.seed,
+            use_tensorboard=cfgs.use_tensorboard,
+            use_wandb=cfgs.use_wandb,
+            config=cfgs,
```

Comment:
if `configs=cfgs`, why specifically passing `seed=cfgs.seed`

Reply:
The argument config is passed to the Logger for the purpose of saving to Weights and Biases. However, if the user does not require this functionality, they may pass None as the config argument. In this sitution, user can only pass `seed`, `output_dir` etc for basic logger use. This provides greater flexibility and eliminates unnecessary dependencies. To accommodate varying needs, another argument can be utilized instead, making the code more modular, adaptable, reusable, and maintainable.